allensdk/model/biophysical/utils.py

```diff
@@ -93,7 +98,7 @@ class Utils(HocUtils):
 
     _log = logging.getLogger(__name__)
 
-    def __init__(self, description):                    
+    def __init__(self, description,axon_type=None):                    
```

Comment:
please write a docstring here. Also, why did you choose for the default to be `None` vs. a descriptive string (since the only alternateive is "stub_axon")?